lab_1/main.py

```diff
@@ -12,7 +14,14 @@ def tokenize(text: str) -> list:
     e.g. text = 'The weather is sunny, the man is happy.'
     --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
     """
-    pass
+    if isinstance(text, str):
+        text = text.lower()
+        tokens = re.sub('[\'\",.!0-9+=\!@#$%^&*()-_]+', '', text).split()
+        return tokens
+    else:
```

Comment:
no need to use else here
if return tokens works, the function will finish its work